internal/scanner/scanner.go

```diff
@@ -74,9 +88,69 @@ func (s *Scanner) ScanNamespace(namespace string) error {
 	// unstructured nested functions might help with that https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured
 	// or the UnstructuredContent() method that returns a map
 
+	for i := range auditableResources {
+		auditResource(&auditableResources[i], &s.resourcesFetcher)
+	}
+
 	return nil
 }
 
+func auditResource(resource *resources.AuditableResources, resourcesFetcher *ResourcesFetcher) {
+	for i := range resource.Policies {
+		url, err := (*resourcesFetcher).GetPolicyServerURLRunningPolicy(context.Background(), &resource.Policies[i])
+		if err != nil {
+			// TODO what's the better thing to do here?
+			fmt.Println(err)
+			continue
+		}
+		for _, resource := range resource.Resources {
+			admissionRequest, err := resources.GenerateAdmissionReview(resource)
+			if err != nil {
+				// TODO what's the better thing to do here?
+				fmt.Println(err)
+				continue
+			}
+
+			auditResponse, err := sendAdmissionReviewToPolicyServer(url, admissionRequest)
+			if err != nil {
+				// TODO what's the better thing to do here?
+				fmt.Println(err)
+				continue
+			}
+			fmt.Println(auditResponse)
+		}
+	}
+}
+
+func sendAdmissionReviewToPolicyServer(url string, admissionRequest *admv1.AdmissionReview) (*admv1.AdmissionReview, error) {
+	payload, err := json.Marshal(admissionRequest)
+
+	if err != nil {
+		return nil, err
+	}
+	// TODO remove the following line and properly configure the certificates
+	http.DefaultTransport.(*http.Transport).TLSClientConfig = &tls.Config{InsecureSkipVerify: true} //nolint
+	res, err := http.Post(url, "application/json", bytes.NewBuffer(payload))                        //nolint
+	if err != nil {
+		return nil, err
+	}
+	defer res.Body.Close()
+	body, err := io.ReadAll(res.Body)
+	if res.StatusCode > 299 {
+		return nil, fmt.Errorf("response failed with status code: %d and\nbody: %s", res.StatusCode, body)
+	}
+	if err != nil {
+		return nil, err
+	}
```

Comment:
This has to be moved before the `if res.StatusCode > 299`. The contents of the body can be used to build the error message on line 140, hence we have to make sure we actually managed to read them.

BTW, I would also wrap up `err` to give some meaningful information:

```golang
return nil, fmt.Errorf("Cannot read body of response: %w", err)
```